R/geom-beeswarm-rast.R

```diff
@@ -0,0 +1,81 @@
+DrawGeomPointRast <- function(data, panel_params, coord, na.rm = FALSE, width=NULL, height=NULL, dpi=300) {
```

Comment:
Looks like this function is completely the same as in `geom_point_rast`. In this case, you don't need to redefine it.